### PR TITLE
Release Google.Cloud.SecretManager.V1 version 1.3.0

### PR DESCRIPTION
**Pull request by jskeet**
Changes in this release:

- [Commit 9bd5f77](https://github.com/googleapis/google-cloud-dotnet/commit/9bd5f77): feat: added expire_time and ttl fields to Secret

### CODE REVIEW

File: azure/AssetTest/.gitignore [hunk 1]
```diff
@@ -0,0 +1,264 @@
+## Ignore Visual Studio temporary files, build results, and
```
Comment (amanda-tarafa) [inline azure/AssetTest/.gitignore:1]: I think this file shouldn't be here.

File: azure/AssetTest/AssetTest.csproj [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (amanda-tarafa) [inline azure/AssetTest/AssetTest.csproj:1]: Or this one. Seems like this Asset testing project crept in this PR somehow.

File: install_functions.sh [hunk 3]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```
Comment (amanda-tarafa) [inline install_functions.sh:1]: An this one as well.

**Review by amanda-tarafa — COMMENTED** [review]
There are some files here that don't belong I believe.
What belongs looks good.

**Comment by jskeet** [comment]
Whoops... thanks, killing this...

**Pull request closed without merging**